src/default_tests.jsonc

```diff
@@ -16,17 +16,18 @@
     // uncomment the second to run NOAA-GFDL-AM4 only for the MJO_prop_amp POD,
     // and likewise for the SM_ET_coupling POD.
     {
-      "CASENAME" : "QBOi.EXP1.AMIP.001",
+      "CASENAME" : "QBOi_CROPPED.EXP1.AMIP.001",
```

Comment:
Remove the changes from the default_tests.jsonc file. We can eventually include the POD in the file, but we need to make sure that we have the  test dataset available on our FTP server, and need to place QBOi_CROPPED in its own case block.